_episodes/13-looking-up-data.md

```diff
@@ -49,7 +49,11 @@ The next exercise demonstrates this two stage process in full.
 >* Give the column a name e.g. "Journal details"
 >* In the expression box you need to write some GREL where the output of the expression is a URL which can be used to retrieve data (the format of the data could be HTML, XML, JSON, or some other text format)
 >
->In this case we are going to use the CrossRef API: [https://github.com/CrossRef/rest-api-doc)](https://github.com/CrossRef/rest-api-doc). Read more about the CrossRef service: [http://www.crossref.org](http://www.crossref.org). Note that the API providers may impose rate limits or have other requirements for using their data, so it's important to check the site's documentation.
+>In this case we are going to use the CrossRef API: [https://github.com/CrossRef/rest-api-doc)](https://github.com/CrossRef/rest-api-doc). Read more about the CrossRef service: [http://www.crossref.org](http://www.crossref.org). Note that API providers may impose rate limits or have other requirements for using their data, so it's important to check the site's documentation. CrossRef, for instance, [asks users](https://github.com/CrossRef/rest-api-doc#etiquette) to "specify a User-Agent header that properly identifies your script or tool and that provides a means of contacting you via email using 'mailto:'." User-agent headers provide administrators with user information that facilitates better administration and moderation of the API, and it is generally good etiquette to include a header with any API request.
+>
+>To edit your User-Agent header:
+>* Click 'Show' (next to 'HTTP headers to be used when fetching URLs'). Note that OpenRefine has already populated the 'User-Agent' field with some information.
+>* At the end of the existing text, add ```; mailto:address@library.edu```, using your own email address. The full User-Agent field should now read ```OpenRefine 3.2 [55c921a]; mailto:address@library.edu```.
```

Comment:
The precise content will depend on the version of OpenRefine used - it would be good to make it clear that the text here will vary

Reply:
Sounds good - let me know if my additional commit (045097c) sufficiently resolves this - thanks!